tests/trainer/test_data_loading.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, num_workers):
     def train_dataloader(self):
         return DataLoader(RandomDataset(32, 64), num_workers=self.num_workers)
 
-    def on_pretrain_routine_start(self):
+    def setup(self, stage=None):
```

Comment:
`on_fit_start` would also work here, right?
It makes more sense because `on_pretrain_routine_start` also only ran for fitting.